akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -215,6 +215,16 @@ object HttpEntity {
     if (contentType == Empty.contentType) Empty
     else HttpEntity.Strict(contentType, data = ByteString.empty)
 
+  {
+    // FIXME: Exists only to fix JavaDSL init problem: #19162. Remove in Akka 3.x when "EMPTY" static fields moved
+    val u = Unsafe.instance
```

Comment:
Is there a reason for having this code duplicated many times? It would seem easier to create a utility for this so that the code becomes a one-liner which is easy to see all occurrences of?